docs/eigenda/networks/mainnet.md

```diff
@@ -4,6 +4,20 @@ sidebar_position: 3
 
 # Mainnet
 
-The EigenDA mainnet is coming soon. Stay tuned [on twitter][1] for updates.
+## Quick Links
 
-[1]: https://twitter.com/eigen_da
+* [AVS Page][2]
+* [Blob Explorer][1]
+* [Deployed Contract Addresses][3]
+
+## Specs
+
+| Property | Value |
+| --- | --- |
+| Disperser Address | `disperser.eigenda.xyz:443` |
+| Batch Confirmation Interval | Every 10 minutes (may vary based on network health) |
+| Max Blob Size | 2 MB |
+
```

Comment:
what about the rate limiting params?

Reply:
There is no default rate limits on mainnet (no default free tier)